Automattic/jetpack
# JITM: change 'setup' to 'set up' in pre-connection JITMs

Fixes #15998

#### Changes proposed in this Pull Request:
* In the pre-connection JITMS, change the word 'setup' to the phrase 'set up'.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* This changes an existing feature.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. On a brand new site, before connecting to WordPress.com, add the following:
`add_filter( 'jetpack_pre_connection_prompt_helpers', '__return_true' );`

2. Verify that the pre-connection JITMs use 'Set up' in the notice text and the button. The JITMs are displayed on the Posts, Widget, and Media pages.

#### Proposed changelog entry for your changes:
* n/a

cc @robertf4 @jeherve - I added this to the 8.6 milestone. It's not a blocker, so if you would prefer to push it to 8.7, that's fine with me.
